runner/src/server/plugins/engine/pageControllers/PageControllerBase.ts

```diff
@@ -376,6 +377,13 @@ export class PageControllerBase {
 
   makeGetRouteHandler() {
     return async (request: HapiRequest, h: HapiResponseToolkit) => {
+      // NOTE: Start pages should live on gov.uk, but this allows prototypes to include signposting about having to log in.
+      if (
+        this.pageDef.controller !== "./pages/start.js" &&
```

Comment:
how will you be using auth at DIT? should a user be auth'd first before they do anything else? 

it might be worth looking at this (scroll down to the basic example) https://hapi.dev/tutorials/auth. My concern is that it's easy to forget if you're adding a new route for example, to make sure the user is redirected to the login page (especially if the team adding route don't use auth) 

this bit:
```
    const validate = (request, user, pass) => {..}
    server.auth.strategy('simple', 'basic', { validate });
```

Reply:
> how will you be using auth at DIT? should a user be auth'd first before they do anything else?

Yes, they should -- to keep things simple the auth strategy kicks in for every single form page (apart from start pages). We need the whole form to be behind a login -- leaving all public pages accessible, as well as the start page so users can be told about the need to sign in. One could flag whether any given form page requires auth, but that would be beyond the scope of this first implementation -- though very much feasible!

We're integrating with our Single Sign-On service to access some user data that we're hoping to use to pre-populate certain fields (name, email).

Reply:
Ah ok, so it would be per form? `runner/src/server/plugins/engine/plugin.ts` might be a better place to put it

Reply:
Yup, that makes sense -- sorted